rakelib/plugin.rake

```diff
@@ -61,6 +66,18 @@ namespace "plugin" do
     task.reenable # Allow this task to be run again
   end
 
+  task "remove-non-oss-plugins" do |task, _|
+    puts("[plugin:remove-non-oss-plugins] Removing non-OSS plugins")
+
+    LogStash::RakeLib::OSS_EXCLUDED_PLUGINS.each do |plugin|
+      remove_plugin(plugin)
+      # gem folder and spec file still stay after removing the plugin
+      FileUtils.rm_r(Dir.glob("#{LogStash::Environment::BUNDLE_DIR}/**/gems/#{plugin}*"))
+      FileUtils.rm_r(Dir.glob("#{LogStash::Environment::BUNDLE_DIR}/**/specifications/#{plugin}*.gemspec"))
```

Comment:
During the plugin installation, we are [copying _local gem_ to jruby/cache](https://github.com/mashhurs/logstash/blob/214fbcddacd2b4a153eccc8e9223174fa4de8c41/lib/pluginmanager/install.rb#L262). We need to also remove the cache when we remove the plugin.